libbeat/processors/config.go

```diff
@@ -21,7 +21,8 @@ import (
 	"github.com/elastic/beats/libbeat/common"
 )
 
-type PluginConfig []map[string]*common.Config
+// PluginConfig represents the list of processors.
+type PluginConfig []*common.Config
```

Comment:
Looking at the comment maybe `Processors` or `List` might be a better name?

Reply:
I didn't want to change it because it's public and used in multiple places in the code. But I agree the name isn't that great.

Reply:
oh yeah its public :(